roles/deploy/tasks/rollbacks.yml

```diff
@@ -9,13 +9,15 @@
     target: "{{ rollback_tmp_path }}"
   become: yes
   become_user: postgres
+  when: db_host == 'localhost'
```

Comment:
So we don't make a backup during deployment?

Reply:
This was totally coupled to dbservers running alongside the webserver so I thought skipping this was a good trade-off. Doing the backup against DO's hosted DB can be explored later on.